src/components/tab-title/tab-title.tsx

```diff
@@ -184,7 +185,9 @@ export class TabTitle implements InteractiveComponent {
   }
 
   componentDidRender(): void {
-    updateHostInteraction(this, true);
+    updateHostInteraction(this, () => {
+      return this.active;
```

Comment:
Added this so that tabindex wasn't conflicting with `updateHostInteraction` utility